src/data/valid/Database-pooling_set-exhaustive.yaml

```diff
@@ -5,8 +5,8 @@ pooling_set:
       - generic:xxx
     description: xxx
     has_input:
-      - generic:xxx
-      - generic:yyy
+      - nmdc:bsm-99-123456
+      - nmdc:bsm-99-987654
     has_output:
       - generic:xxx
```

Comment:
Pooling has_output range needs to be updated to ProcessedSample and example data updated.